src/dotty/tools/dotc/core/Contexts.scala

```diff
@@ -68,6 +69,9 @@ object Contexts {
     /** The context base at the root */
     val base: ContextBase
 
+    /** Documentation base */
+    def getDocbase = property(DocContext)
```

Comment:
Can we move this out of Context? The design should be ideally one of a plugin. That is, there is a context property, but the `Contexts` object knows nothing about it. I would do it more like we model `defn`: Put in the main Doc object:
    
     val DocBase = ...
     def getDocBase(implicit ctx: Context) = ctx.property(DocBase)